cpp/src/io/orc/reader_impl.cu

```diff
@@ -1334,11 +1332,7 @@ table_with_metadata reader::impl::read(uint64_t skip_rows,
           }
         });
 
-      if (buff_data.size()) {
-        auto const dev_buff_data = cudf::detail::make_device_uvector_async(
-          buff_data, _stream, rmm::mr::get_current_device_resource());
-        generate_offsets_for_list(dev_buff_data, _stream);
-      }
+      if (buff_data.size()) { generate_offsets_for_list(buff_data, _stream); }
```

Comment:
```suggestion
      if (buff_data.empty()) { generate_offsets_for_list(buff_data, _stream); }
```

Reply:
This suggestion inverts the condition. We could change it to:
```suggestion
      if (buff_data.size() > 0) { generate_offsets_for_list(buff_data, _stream); }
```
or 
```suggestion
      if (!buff_data.empty()) { generate_offsets_for_list(buff_data, _stream); }
```
Do you have a preference?

Reply:
My vote is for `not buff_data.empty()` :)

Reply:
I thought about including that as another option too :D I can do that.